app/views/products/createrecurringdepositproduct.html

```diff
@@ -214,6 +214,26 @@
                         </div>
                     </div>
 
+                    <div class="form-group">
+                        <!-- BEGIN "Deposit every" -->
+                        <label class="control-label col-sm-2">{{ 'label.input.depositEveryFrequency' | translate
+                            }}</label>
+                        <div class="col-sm-2">
+                            <input id="depositEveryFrequency" type="text" class="form-control"
+                                ng-model="formData.depositEveryFrequency">
+                        </div>
+                        <div class="col-sm-2">
+                            <select id="depositEveryFrequencyType" ng-model="formData.depositEveryFrequencyType"
+                                    class="form-control"
+                                    ng-options="type.id as type.value for type in product.depositEveryFrequencyTypeOptions"
+                                    ng-change="depositproduct.depositEveryFrequencyOption.value = formValue(product.depositEveryFrequencyTypeOptions,formData.depositEveryFrequencyType)"
```

Comment:
`depositproduct.depositEveryFrequencyOption.value` Should be changed to `depositproduct.depositEveryFrequencyOption`.  The preview step should also display the field input. Take a look at the code in the preview step, you will understand how to make this input visible over there. It is similar to the way we display it in `viewreccuringdepositproduct.html`